### PR TITLE
Cleanup Scene: remove some unpythonic or niche methods

### PR DESCRIPTION
**Pull request by leotrs**
## Motivation
All the removed methods seem either very niche, or outright useless.

## Testing Status
Local pass.

## Further Comments
Strictly speaking, these are all breaking changes, but I think it's safe to just remove them. Perhaps with the exception of Scene.get_mobjects, I don't think anyone is using these.

### CODE REVIEW

**Review by huguesdevimeux — APPROVED** [review]
SGTM ! 

(it's getting prettier days after days!)

File: manim/scene/scene.py [hunk 1]
```diff
@@ -223,15 +222,6 @@ def add(self, *mobjects):
         self.mobjects += mobjects
         return self
 
-    def add_mobjects_among(self, values):
```
Comment (kolibril13) [inline manim/scene/scene.py:226]: I did not know this method existed.
Probably something like:
```python
class ABC(Scene):
    def construct(self):
        dot = Dot().set_color(GREEN)
        cric= Circle()
        sq= Square()
        self.add_mobjects_among(locals().values())
```
could be useful in some situations, but I am also fine with removing these.

Comment (leotrs) [reply manim/scene/scene.py:226]: This is super unpythonic, and I'm very much against maintaining something like this.

Comment (kolibril13) [reply manim/scene/scene.py:226]: okay!

**Review by kolibril13 — APPROVED** [review]

**Pull request merged**